src/app/ApplicationTemplate.Client/ApiEndpoints/Authentication/AuthenticationToken.cs

```diff
@@ -4,9 +4,9 @@
 
 namespace ApplicationTemplate.Client;
 
-public record AuthenticationToken
+public class AuthenticationToken
```

Comment:
since it's not a record, should it be added to the class for serialization? maybe it's already done

Reply:
it is already in the serialization file